src/Uno.Extensions.Core.Generators/buildTransitive/Uno.Extensions.Core.props

```diff
@@ -9,17 +9,4 @@
 		<CompilerVisibleProperty Include="UnoExtensionsGeneration_DisableIsExternalInit" />
 		<CompilerVisibleProperty Include="UnoExtensionsGeneration_DisableModuleInitializerAttribute" />
 	</ItemGroup>
-
-	<Target Name="_InjectUnoExtensionsCoreGenerators" BeforeTargets="BeforeCompile;MarkupCompilePass1">
-		<ItemGroup>
-			<Analyzer
-				Include="$(MSBuildThisFileDirectory)..\bin\Uno.Extensions.Core.Generators\$(Configuration)\netstandard2.0\Uno.Extensions.Core.Generators.dll"
-				Condition="Exists('$(MSBuildThisFileDirectory)..\bin')" />
-		</ItemGroup>
-	</Target>
-	<ItemGroup>
-		<Analyzer
-			Include="$(MSBuildThisFileDirectory)..\tools\Uno.Extensions.Core.Generators.dll"
-			Condition="Exists('$(MSBuildThisFileDirectory)..\tools')" />
```

Comment:
This is done automatically by the .NET SDK if the generator is packed correctly in `analyzers/dotnet/cs`

Docs: https://learn.microsoft.com/en-us/nuget/guides/analyzers-conventions